README.md

```diff
@@ -0,0 +1,184 @@
+6. Write SQL queries with Active Record
+7. Use preprocessed HTML file with embedded Ruby code.
+8. Use layouts and templates for shared content.
+9. Build a web app that requires the user to log in.
+10. Use devise gem for authentication.
+11. Limit access to web app resources based on authorization rules.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+ 
+
+- **Create a budget app**
+- **register and log in, so that the data is private to them.** 
+- **introduce new transactions associated with a category.**
+- **see the money spent on each category.**
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+
+- [Live Demo Link]()
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+**To get a local copy up and running, follow these steps.**
+1. Open terminal or cmd on you local machine and navigate to your desired directory.
+2. **Run**    *'git clone clone https://github.com/ecekpo/Budget-app.git'*
+3. **Run**   *'code .'*
+
+### Prerequisites
+
+
+In order to run this project you need **[describe prerequisites]**:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/ecekpo/Budget-app.git
+  cd Budget_app
+```
+
+### Install
+
+Install this project with **[package manager]**:
+
+```sh
+  cd Budget_app
+  gem install
+  rails db:migrate
+  rails db:migrate RAILS_ENV=test
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Author
+
+👤 **Emem Ekpo**
+
+* GitHub: [@githubhandle](https://github.com/emem-ekpo)
+* Twitter: [@twitterhandle](https://twitter.com/ememcookey)
+* LinkedIn: [@LinkedIn](https://www.linkedin.com/in/emem-ekpo)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features of this project.
+
+- [ ] **User Authentication**
+- [ ] **CRUD Operations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Credit
+   I give credit to the original design idea by Gregoire Vella on Behance.
+
```

Comment:
- [ ] [REQUIRED] The README contains an "Acknowledgements" that give appropriate credit, However, I will suggest you add the name [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) as the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). This will give more description of the author.

![221359671-d46e3f22-91a6-4fa1-ae39-e3d39e6f985e](https://user-images.githubusercontent.com/63915024/221377751-689733b4-d895-4c99-b54a-8ac001353320.png)